charts/linkerd-control-plane/values.yaml

```diff
@@ -51,6 +47,13 @@ controller:
   podDisruptionBudget:
     # -- Maximum number of pods that can be unavailable during disruption
     maxUnavailable: 1
+  # Configures tracing in the controllers and how traces are exported
+  tracing:
+    # -- Enables trace collection and export in the proxy
+    enable: false
```

Comment:
any point to having this boolean vs just looking at if collector.endpoints is empty?

Reply:
I can't remember either :laughing: 
Looking through #13994, there was discussion around not using boolean values, and validating that `tracing.enable` is consistent with the contents of `collector.endpoint` in the proxy config below. Anyways, relying on `collector.endpoints` emptiness sounds like a simpler approach to me too, but would be good to use the same approach here and in the proxy section.